mmdet/ops/nms/nms_wrapper.py

```diff
@@ -136,10 +136,14 @@ def batched_nms(bboxes, scores, inds, nms_cfg):
     Returns:
         tuple: kept bboxes and indice.
     """
-    max_coordinate = bboxes.max()
-    offsets = inds.to(bboxes) * (max_coordinate + 1)
-    bboxes_for_nms = bboxes + offsets[:, None]
     nms_cfg_ = nms_cfg.copy()
+    class_agnostic = nms_cfg_.pop('class_agnostic', False)
```

Comment:
I suggest adding `class_agnostic` as an argument of `batched_nms()`, with the default value False.

Reply:
So `class_agnostic = nms_cfg_.pop('class_agnostic', class_agnostic)` where `class_agnostic` is an argument of `batched_nms()`?
Or would you get rid of putting it in the `nms_cfg`? Because I would like to keep it there as well.